rapid7_insightidr/help.md

```diff
@@ -552,6 +648,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 2.0.0 - Update to refactor advanced query. Split Advanced Query into two new actions Advanced Query on Log and Advanced Query on Log Set
```

Comment:
```suggestion
* 2.0.0 - Refactor and split Advanced Query into two new actions Advanced Query on Log and Advanced Query on Log Set
```

Reply:
Update to is part of the style guide BTW.

Reply:
Yeah, wanted to consolidate it into a single sentence for readability